content/chapters/compute/lab/support/page-faults/page_faults.c

```diff
@@ -0,0 +1,87 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```
The `/* ... */` format is only meant for `.h` files.